How-To/Connect-to-AWS-Resources.md

```diff
@@ -105,12 +105,14 @@ Using python, the `pyathena` package is an excellent option for ingesting data f
     import pandas
     from pyathena import connect
     from pyathena.pandas.util import as_pandas
+    from pyathena.pandas.cursor import PandasCursor
 
     # Connect to Athena
     conn = connect(
-        s3_staging_dir=os.getenv("AWS_ATHENA_S3_STAGING_DIR"),
+        s3_staging_dir=os.getenv("AWS_ATHENA_S3_STAGING_DIR") + "/",
```

Comment:
**issue:** Add some context about why the `/` is necessary please. And a short explanation of the unload option with a link to the docs.